azurelinuxagent/common/cgroupapi.py

```diff
@@ -368,13 +366,13 @@ def start_extension_command(self, extension_name, command, timeout, shell, cwd,
             finally:
                 with self._systemd_run_commands_lock:
                     self._systemd_run_commands.remove(process.pid)
-        except ExtensionError as e:  # pylint: disable=C0103
+        except ExtensionError as e:
             # The extension didn't terminate successfully. Determine whether it was due to systemd errors or
             # extension errors.
             systemd_failure = self._is_systemd_failure(scope, stderr)
             process_output = read_output(stdout, stderr)
 
-            if not systemd_failure:  # pylint: disable=R1720
+            if not systemd_failure:
```

Comment:
I'm not sure that `R1720=no-else-raise` is the same type of warning as the others here.

This one is referring to the `else` below this `if`, right? It might be clearer to simply remove the `else`.

Reply:
This PR is not attempting to fix any warnings, it is simply suppressing warnings we do not care about globally instead of using inline suppressions.

Yes, the code can be written without the else, do feel free to address those in a separate PR.

Reply:
As per the offline discussion, I think it's fine to leave these as is! My thoughts are that it would be a nice to have, but I don't think a PR should be blocked for not conforming to this style.